python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -575,17 +572,33 @@ def current_in_flight_tasks(self) -> int:
         )
 
     def scale_up(self, num_actors: int) -> int:
+        logger.info(
+            f"Scaling up actor pool by {num_actors} ("
+            f"running={self.num_running_actors()}, "
+            f"pending={self.num_pending_actors()}, "
+            f"restarting={self.num_restarting_actors()})"
+        )
+
```

Comment:
We autoscale actors as much as twice a second, so I'm worried these info logs might get spammy. Would debug make sense instead?

Reply:
There actually should be no bouncing back and forth in autoscaler (it just doesn't really make sense)